server/portal/settings/settings.py

```diff
@@ -91,6 +91,7 @@ def gettext(s): return s  # noqa:E731
     'portal.apps.system_monitor',
     'portal.apps.projects',
     'portal.apps.system_creation',
+    'portal.apps.public_data'
```

Comment:
can you add an example entry into `_PORTAL_DATAFILES_STORAGE_SYSTEMS` in `settings_secret.example.py`?